### PR TITLE
DOC-4321: Reorganize all sections and develop Domain

### PR DESCRIPTION
**Pull request by hghorbel**

### CODE REVIEW

File: docs/Gamification.rst [hunk 1]
```diff
@@ -22,32 +22,18 @@ When employees are engaged, they participate more on your platform by posting, c
 Domains
 =======================
 
-.. _Badges:
+Domains are the categorization of all behaviors of the platform users. 
+Each domain had a different context and specific applications to organize all rules and badges.
```
Comment (plamarque) [inline docs/Gamification.rst]:
```suggestion
Each domain represents a different context and specific applications to organize all rules and badges.
```

File: docs/Gamification.rst [hunk 2]
```diff
@@ -22,32 +22,18 @@ When employees are engaged, they participate more on your platform by posting, c
...
 
-=======================
-Badges
-=======================
+The following are the default domains that made sense in the context of the platform environment:
```
Comment (plamarque) [inline docs/Gamification.rst]:
```suggestion
The following are the default domains that come with eXo Platform: 
```

File: docs/Gamification.rst [hunk 3]
```diff
@@ -22,32 +22,18 @@ When employees are engaged, they participate more on your platform by posting, c
...
-
-In the Leaderboard, the TOP 10 of employees having higher gained points, and every employee is presented by its Full Name, avatar and gained points.
-
-An information of your rank is displayed just below the top 10, if you are not on it. 
-
-You can also load more 10 employees by clicking on Load more button just below the Leaderboard.
-
-|image10|
-When you mouse over an employee in the leaderboard, a detailed pie chart is displayed next to him displaying all pourcentages of gained points based on the domain.
+.. note:: You can add, edit or remove domain using :ref: `Manage Domain <Manage Domain>` in Gamification Administration .
```
Comment (plamarque) [inline docs/Gamification.rst]:
```suggestion
.. note:: You can add, edit or remove domains using :ref: `Manage Domain <Manage Domain>` in Gamification Administration .
```

File: docs/Gamification.rst [hunk 4]
```diff
@@ -100,6 +86,29 @@ When you mouse over a badge, its information appear in a pop up:
...
 |image8|
 
+
+.. _Leaderboard:
+
+=============
+Leaderboard 
+=============
+
+Leaderboards will give you a snapshot of your gained points progress, and of other collaborators in your platform to encourage them to gain more points.
```
Comment (plamarque) [inline docs/Gamification.rst]:
```suggestion
Leaderboards display a snapshot of how your earned points compare to other users. It's a friendly competition designed to encourage everyone to earn more points. 
```

File: docs/Gamification.rst [hunk 5]
```diff
@@ -100,6 +86,29 @@ When you mouse over a badge, its information appear in a pop up:
...
+
+=============
+Leaderboard 
+=============
+
+Leaderboards will give you a snapshot of your gained points progress, and of other collaborators in your platform to encourage them to gain more points. 
+
+|image9|
+
+Place in Leaderboard is determined by number of points earned by period : Week, Month or All times and filtered by selected domain.
```
Comment (plamarque) [inline docs/Gamification.rst]:
```suggestion
The position in the leaderboard is determined by the number of points earned over the period : Week, Month or All times and filtered by selected domain.
```

File: docs/Gamification.rst [hunk 6]
```diff
@@ -100,6 +86,29 @@ When you mouse over a badge, its information appear in a pop up:
...
+
+In the Leaderboard, the TOP 10 of employees having higher gained points, and every employee is presented by its Full Name, avatar and gained points.
```
Comment (plamarque) [inline docs/Gamification.rst]:
```suggestion
In the Leaderboard, the top 10 of users having earned most points, and every user is presented with their full name, profile picture and earned points.
```

File: docs/Gamification.rst [hunk 7]
```diff
@@ -100,6 +86,29 @@ When you mouse over a badge, its information appear in a pop up:
...
+
+An information of your rank is displayed just below the top 10, if you are not on it.
```
Comment (plamarque) [inline docs/Gamification.rst]:
```suggestion
Your rank is displayed just below the top 10, if you are not in it. 
```

File: docs/Gamification.rst [hunk 8]
```diff
@@ -100,6 +86,29 @@ When you mouse over a badge, its information appear in a pop up:
...
+
+|image9|
+
+Place in Leaderboard is determined by number of points earned by period : Week, Month or All times and filtered by selected domain.
+
+In the Leaderboard, the TOP 10 of employees having higher gained points, and every employee is presented by its Full Name, avatar and gained points.
+
+An information of your rank is displayed just below the top 10, if you are not on it. 
+
+You can also load more 10 employees by clicking on Load more button just below the Leaderboard.
```
Comment (plamarque) [inline docs/Gamification.rst]:
```suggestion
You can also load more 10 users by clicking on Load more button just below the Leaderboard.
```

File: docs/Gamification.rst [hunk 9]
```diff
@@ -100,6 +86,29 @@ When you mouse over a badge, its information appear in a pop up:
...
+
+|image10|
+When you mouse over an employee in the leaderboard, a detailed pie chart is displayed next to him displaying all pourcentages of gained points based on the domain.
```
Comment (plamarque) [inline docs/Gamification.rst]:
```suggestion
When you mouse over a user in the leaderboard, a pie chart is displayed next to him to detail all proportions of points they earned in each domain.
```

**Review by plamarque — CHANGES_REQUESTED** [review]
proofreading with substantial rewrite to enhance fluidity.

**Comment by hghorbel** [comment]
Update Done.

File: docs/Gamification.rst [hunk 10]
```diff
@@ -22,32 +22,18 @@ When employees are engaged, they participate more on your platform by posting, c
...
-=============
-Leaderboard 
-=============
+- Social: living together in organized communities, typically with different opinion shared on activities, by comments, likes; in your profile or created spaces. 
 
-Leaderboards will give you a snapshot of your gained points progress, and of other collaborators in your platform to encourage them to gain more points. 
+- Team work : The combined action of a group, especially when effective and efficient, and it's ensured using Tasks management.
 
-|image9|
+- Knowledge : Facts, informations, and skills acquired through experiences using Forum, Wiki and Documents applications.
```
Comment (mzorai) [inline docs/Gamification.rst]:
```suggestion
- Knowledge: Facts, information, and skills acquired through experiences using Forum, Wiki and Documents applications.
```

File: docs/Gamification.rst [hunk 11]
```diff
@@ -100,6 +86,29 @@ When you mouse over a badge, its information appear in a pop up:
...
+
+|image9|
+
+The position in the leaderboard is determined by the number of points earned over the period : Week, Month or All times and filtered by selected domain.
+
+In the Leaderboard, the top 10 of users having earned most points, and every user is presented with their full name, profile picture and earned points.
+
+Your rank is displayed just below the top 10, if you are not in it. 
+
+You can also load more 10 users by clicking on Load more button just below the Leaderboard.
```
Comment (mzorai) [inline docs/Gamification.rst]:
```suggestion
You can also load 10 more users by clicking on Load more button just below the Leaderboard.
```

**Review by mzorai — CHANGES_REQUESTED** [review]

**Review by mzorai — APPROVED** [review]

File: docs/Gamification.rst [hunk 12]
```diff
@@ -22,32 +22,18 @@ When employees are engaged, they participate more on your platform by posting, c
...
-
-In the Leaderboard, the TOP 10 of employees having higher gained points, and every employee is presented by its Full Name, avatar and gained points.
-
-An information of your rank is displayed just below the top 10, if you are not on it. 
-
-You can also load more 10 employees by clicking on Load more button just below the Leaderboard.
-
-|image10|
-When you mouse over an employee in the leaderboard, a detailed pie chart is displayed next to him displaying all pourcentages of gained points based on the domain.
+.. note:: You can add, edit or remove domains using :ref: `Manage Domain <Manage Domain>` in Gamification Administration .
```
Comment (plamarque) [inline docs/Gamification.rst]:
```suggestion
.. note:: You can add, edit or remove your own domains using :ref: `Manage Domain <Manage Domain>` in Gamification Administration .
```

File: docs/Gamification.rst [hunk 13]
```diff
@@ -100,6 +86,29 @@ When you mouse over a badge, its information appear in a pop up:
...
 |image8|
 
+
+.. _Leaderboard:
+
+=============
+Leaderboard 
+=============
+
+Leaderboards display a snapshot of how your earned points compare to other users. It's a friendly competition designed to encourage everyone to earn more points.
```
Comment (plamarque) [inline docs/Gamification.rst]:
```suggestion
Leaderboards display a snapshot of users with most points. You always see your own rank so you can compare to others in various domains. It's a friendly competition designed to encourage everyone to engage in various activities by earning points. 
```

File: docs/Gamification.rst [hunk 14]
```diff
@@ -100,6 +86,29 @@ When you mouse over a badge, its information appear in a pop up:
...
+The position in the leaderboard is determined by the number of points earned over the period : Week, Month or All times and filtered by selected domain.
+
+In the Leaderboard, the top 10 of users having earned most points, and every user is presented with their full name, profile picture and earned points.
+
+Your rank is displayed just below the top 10, if you are not in it. 
+
+You can also load 10 more users by clicking on Load more button just below the Leaderboard.
+
+|image10|
+When you mouse over a user in the leaderboard, a pie chart is displayed next to him to detail all proportions of points they earned in each domain.
```
Comment (plamarque) [inline docs/Gamification.rst]:
```suggestion
When you mouse over a user in the leaderboard, a pie chart is displayed beside with details of the distribution of points across domains. Hence you can discover what are their sweet spot domains and learn from their behaviours.
```

**Review by plamarque — CHANGES_REQUESTED** [review]
More optimizations requested

**Comment by hghorbel** [comment]
Done.

File: docs/Gamification.rst [hunk 15]
```diff
@@ -100,6 +86,29 @@ When you mouse over a badge, its information appear in a pop up:
...
 |image8|
 
+
+.. _Leaderboard:
+
+=============
+Leaderboard 
+=============
+
+Leaderboards display a snapshot of users with most points. You always see your own rank so you can compare to others in various domains. It's a friendly competition designed to encourage everyone to engage in various activities by earning points.
```
Comment (plamarque) [inline docs/Gamification.rst]:
```suggestion
Leaderboards display a snapshot of users with most points. You always see your own rank so you can compare yourself to others in various domains. It's a friendly competition designed to encourage everyone to engage in various activities by earning points. 
```

File: docs/Gamification.rst [hunk 16]
```diff
@@ -100,6 +86,29 @@ When you mouse over a badge, its information appear in a pop up:
...
+
+=============
+Leaderboard 
+=============
+
+Leaderboards display a snapshot of users with most points. You always see your own rank so you can compare to others in various domains. It's a friendly competition designed to encourage everyone to engage in various activities by earning points. 
+
+|image9|
+
+The position in the leaderboard is determined by the number of points earned over the period : Week, Month or All times and filtered by selected domain.
```
Comment (plamarque) [inline docs/Gamification.rst]:
```suggestion
The position in the leaderboard is determined by the number of points earned over the period : Week, Month or All times and is filtered by selected domain.
```

File: docs/Gamification.rst [hunk 17]
```diff
@@ -100,6 +86,29 @@ When you mouse over a badge, its information appear in a pop up:
...
+
+In the Leaderboard, the top 10 of users having earned most points, and every user is presented with their full name, profile picture and earned points.
```
Comment (plamarque) [inline docs/Gamification.rst]:
```suggestion
In the Leaderboard, the top 10 users with higher scores over the period are with their full name, profile picture and point scores.
```

File: docs/Gamification.rst [hunk 18]
```diff
@@ -100,6 +86,29 @@ When you mouse over a badge, its information appear in a pop up:
...
+
+Your rank is displayed just below the top 10, if you are not in it. 
+
+You can also load 10 more users by clicking on Load more button just below the Leaderboard.
```
Comment (plamarque) [inline docs/Gamification.rst]:
```suggestion
You can also navigate in the ranks by clicking on Load more button to load 10 more entries.
```

File: docs/Gamification.rst [hunk 19]
```diff
@@ -100,6 +86,29 @@ When you mouse over a badge, its information appear in a pop up:
...
+
+|image10|
+When you mouse over a user in the leaderboard, a pie chart is displayed beside with details of the distribution of points across domains. Hence you can discover what are their sweet spot domains and learn from their behaviours.
```
Comment (plamarque) [inline docs/Gamification.rst]:
```suggestion
When you mouse over a user in the leaderboard, a pie chart is displayed to illustrate the distribution of points across domains. Hence you can discover what are their sweet spot domains and learn from their behaviors.
```

**Review by plamarque — CHANGES_REQUESTED** [review]
2nd pass proofreading and suggested optimizations

File: docs/Gamification.rst [hunk 20]
```diff
@@ -22,32 +22,18 @@ When employees are engaged, they participate more on your platform by posting, c
 Domains
 =======================
 
-.. _Badges:
+Domains are the categorization of all behaviors of the platform users.
```
Comment (plamarque) [inline docs/Gamification.rst]:
```suggestion
Domains are used to categorize all behaviors of the platform users. 
```

File: docs/Gamification.rst [hunk 21]
```diff
@@ -22,32 +22,18 @@ When employees are engaged, they participate more on your platform by posting, c
 Domains
 =======================
 
-.. _Badges:
+Domains are the categorization of all behaviors of the platform users. 
+Each domain represents a different context and specific applications to organize all rules and badges.
```
Comment (plamarque) [inline docs/Gamification.rst]:
```suggestion
Each domain contains rules and badges related to a different context and usage of specific applications.
```

File: docs/Gamification.rst [hunk 22]
```diff
@@ -100,6 +86,29 @@ When you mouse over a badge, its information appear in a pop up:
...
+Leaderboard 
+=============
+
+Leaderboards display a snapshot of users with most points. You always see your own rank so you can compare yourself to others in various domains. It's a friendly competition designed to encourage everyone to engage in various activities by earning points. 
+
+|image9|
+
+The position in the leaderboard is determined by the number of points earned over the period : Week, Month or All times and is filtered by selected domain.
+
+In the Leaderboard, the top 10 users with higher scores over the period are with their full name, profile picture and point scores.
```
Comment (plamarque) [inline docs/Gamification.rst]:
```suggestion
In the Leaderboard, the top 10 users with the highest scores over the period are displayed with their full name, profile picture and score (points gathered).
```

**Review by plamarque — CHANGES_REQUESTED** [review]
another pass for proofreading

**Comment by hghorbel** [comment]
Changes DONE.

**Comment by hghorbel** [comment]
@plamarque waiting for your approval to merge.

**Review by plamarque — APPROVED** [review]
LGTM

**Pull request merged**